### PR TITLE
MealPlannerClasses Initial Commit 18-02-2020

### PR DESCRIPTION
**Pull request by hpowell20**

### CODE REVIEW

File: classes.py [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+		""" 
+		Constructor for initialising the User Object
+		Constructor initialises the 	loggedIn status, 	initial value = False
+										UserID, 			initial value = 0
+
+
+		**kwargs: Additional keyword arguments to pass to a superclass constructor 
+		"""
+
+		self.LoggedIn = LoggedIn_
```
Comment (hpowell20) [inline classes.py]: This field will be handled through Cognito; if an authenticated session is returned they are logged in.

File: classes.py [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+		self.UserID =  UserID_
```
Comment (hpowell20) [inline classes.py]: Primary key?

File: classes.py [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+
+######################################## CLASS USERPROFILE ###############################
+class UserProfile:
+''' 
+The UserProfile is a class which contains information pertaining to a User with known ID.
+Every user should have corresponding UserProfile object. 
+
+-------To do: add UserProfile Object to the User Class.
+
+-------Question: I wonder if it is beneficial to combine to the two classes into one?
```
Comment (hpowell20) [inline classes.py]: Good catch.  This should be one table.

File: classes.py [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+
+Notes:
+
+'''
+	def __init__(self,  userID_= 0, date_ = 0, arrayOfDislikes_ = [], arrayOfWeeklyPlanID_ = [], *args, **kwargs):
+		'''
+		Constructor for initialising the UserProfile.
+		-----	userID_ corresponds to the userID and is defaulted to value 0.
+		-----	date_ corresponds to the date??? and is defaulted to value 0. ******see To do!
```
Comment (hpowell20) [inline classes.py]: Which date are you planning on using?

File: classes.py [hunk 5]
```diff
@@ -0,0 +1,416 @@
+'''
```
Comment (hpowell20) [inline classes.py]: For all tables look at adding in created_at, created_by, updated_at, and updated_by

**Comment by vagmi** [comment]
Since we are going the postgraphile route, I am closing this pull request.

**Pull request closed without merging**